### DISCUSSION TITLE
[Bug]: cpf_or_cnpj field is not saving properly on the database

### DISCUSSION THREAD

**Discussion by BRGustavoRibeiro** [leandrocfe/filament-ptbr-form-fields#58] [Q&A]
### What happened?

I expected _cpf_or_cnpj_ field to save data on the database without the mask, and the `dehydrateMask()` method should be available in case I wanted to save the info with the mask.

### How to reproduce the bug

Fill a _cpf_or_cnpj_ field. Save it. Check it on the database - it is saved as `111.111.111-11` instead of `11111111111`

### Package Version

3.0.1

### PHP Version

8.2.0

### Laravel Version

10.3.3

### Which operating systems does with happen with?

Windows

**Comment by bianchi** [comment]
Tenho o mesmo problema, resolvi adicionando no meu EditCustomer o seguinte:

```
protected function mutateFormDataBeforeSave(array $data): array
{
        $data['document_number'] = preg_replace('/\D/', '', $data['document_number']);

        return $data;
}
```

**Comment by lucasff** [comment]
O ideal seria fazer isso no Model, não?
